packages/framer-motion/src/animation/waapi/create-accelerated-animation.ts

```diff
@@ -69,7 +72,8 @@ export function createAcceleratedAnimation(
          */
         let t = 0
         while (!state.done && t < 20000) {
-            state = sampleAnimation.sample(t, true)
+            // TODO Reinstate sample
+            state = sampleAnimation.sample(t)
```

Comment:
I don't understand this TODO, could you make it a little bit more elaborate?

Reply:
Actually I can remove - this was from before I redid `sample` to take advantage of the stateless animation